### PR TITLE
Add use case 5 for capital cities

### PR DESCRIPTION
**Pull request by ellie2009**

### CODE REVIEW

File: use-cases/use-case5.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+**USE CASE: 4 Produce a report on population statistics**
```
Comment (NathanPr03) [inline use-cases/use-case5.md:1]: This has the wrong number

File: use-cases/use-case5.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+3. User extracts information for all the capital cities in a region organised by largest to smallest.
+
+4. User extracts information for the top N populated capital cities in the world where N is provided by the user.
+
+5. User extracts information for the top N populated capital cities in a continent where N is provided by the user.
+
+6. User extracts information for the top N populated capital cities in a region where N is provided by the user.
+
+7.User provides report to organisation.
```
Comment (NathanPr03) [inline use-cases/use-case5.md:50]: Whitespace

File: use-cases/use-case5.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+4, 5, 6 User provides a negative number when querying the top number populated capital cities
+
+**SUB-VARIATIONS**
+
+None.
+
+**SCHEDULE**
+
+**DUE DATE:** Release 1.0
```
Comment (NathanPr03) [inline use-cases/use-case5.md:63]: Wrong release number

File: use-cases/use-case5.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+**USE CASE: 4 Produce a report on population statistics**
+
+**CHARACTERISTIC INFORMATION**
+
+**Goal in Context**
+
+As a user I want to retrieve reports on capital cities and their populations so that I can support the organisation’s business goals
+**Scope**
```
Comment (NathanPr03) [inline use-cases/use-case5.md:8]: Whitespace

**Review by NathanPr03 — CHANGES_REQUESTED** [review]

**Review by Nathan03nable — APPROVED** [review]

**Pull request merged**